frameworks/helloworld/tests/test_placement.py

```diff
@@ -24,6 +24,48 @@ def configure_package(configure_security):
         sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
 
 
+@pytest.mark.dcos_min_version("1.12")
+@pytest.mark.sanity
+def test_scheduler_task_placement_by_marathon():
```

Comment:
This is the only new method. Rest of the diff in this file is restructuring (I reordered methods so that test_* comes first followed by helpers).